kit/src/app.css

```diff
@@ -0,0 +1,422 @@
+				0px 2px 6px rgba(0, 0, 0, 0.04), 0px 0px 1px rgba(0, 0, 0, 0.04);
+		}
+		.shadow-alternate-xl {
+			box-shadow: 0px 24px 32px rgba(0, 0, 0, 0.04),
+				0px 16px 24px rgba(0, 0, 0, 0.04), 0px 4px 8px rgba(0, 0, 0, 0.04),
+				0px 0px 1px rgba(0, 0, 0, 0.04);
+		}
+	}
+	@variants responsive {
+		.from-gray-50-to-white {
+			@apply from-gray-50 to-white dark:from-gray-925 dark:to-gray-950;
+		}
+
+		.from-gray-100-to-white {
+			@apply from-gray-100 to-white dark:from-gray-925 dark:to-gray-925;
+		}
+	}
+	.scroll-smooth {
+		scroll-behavior: smooth;
+	}
+}
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+
+
+/** Prose **/
+
+.prose iframe {
+	@apply my-10 rounded-lg overflow-hidden;
+}
+
+.prose .course-tip {
+	@apply px-5 py-3.5 mb-3 mt-10 relative before:border-l-2 before:absolute before:inset-y-0 before:left-0;
+}
+
+.prose .course-tip p {
+	@apply !m-0;
+}
+
+.prose code {
+	@apply font-normal;
+}
+
+.prose code,
+.prose strong {
+	@apply text-gray-800;
+}
+
+.prose .course-tip code,
+.prose .course-tip a {
+	@apply text-green-800 dark:text-green-600;
+}
+
+.prose .course-tip.course-tip-orange code,
+.prose .course-tip.course-tip-orange a {
+	@apply text-orange-800 dark:text-orange-600;
+}
+
+.prose .course-tip ul {
+	@apply last:mb-0;
+}
+
+/* Remove the ticks from prose class everywhere */
+.prose code::after,
+.prose code::before {
+	content: '';
+}
+
+/* Prose Doc */
+
+.prose.prose-doc iframe {
+	@apply max-w-full;
+}
+
+.prose.prose-doc .course-tip {
+	@apply mt-6;
+}
+
+.prose.prose-doc .code-block {
+	@apply -mx-3 md:mx-0 2xl:-mx-4 text-smd md:text-base;
+}
+
+.prose.prose-doc .code-block pre {
+	@apply border border-gray-100;
+}
+
+.prose.prose-doc > p > em {
+	@apply text-gray-500;
+}
+
+.prose.prose-doc h1 {
+	@apply text-2xl text-gray-900;
+}
+
+.prose.prose-doc h2 {
+	@apply text-xl text-gray-800;
+}
+
+.prose.prose-doc h3 {
+	@apply text-lg text-gray-800;
+}
+
+.prose.prose-doc h5 {
+	@apply font-semibold text-gray-800;
+}
+
+.prose .prose.prose-doc p {
+	@apply my-7;
+}
+
+.prose.prose-doc img {
+	@apply !max-w-full xl:!max-w-xl;
+}
+
+.prose.prose-doc img[src^="https://img.shields.io/"]
+{
+	@apply m-0 mb-1;
+}
+
+.prose.prose-doc table {
+	@apply block overflow-auto;
+}
+
+/* Prose Doc Docstring */
+
+#docstring-tooltip code {
+	@apply text-sm;
+}
+
+.prose.prose-doc .docstring {
+	@apply border-l-2 border-t-2 pl-4 pt-3.5 border-gray-100 rounded-tl-xl mb-6;
+}
+
+.prose.prose-doc .docstring {
+	@apply mt-8;
+}
+
+.prose.prose-doc .docstring .course-tip {
+	@apply text-base my-6;
+}
+
+.prose.prose-doc > .docstring pre {
+	@apply last:mb-6;
+}
+
+.prose.prose-doc .docstring *,
+.prose.prose-doc .docstring * pre {
+	@apply last:mb-0;
+}
+
+/*.prose.prose-doc .docstring .text-base p {
+	 @apply text-base; 
+}
+
+.prose.prose-doc .docstring li.text-base,
+.prose.prose-doc .docstring .text-base li {
+	@apply my-3;
+}
+
+.prose.prose-doc .docstring .text-base code {
+	@apply text-sm;
+}
```

Comment:
Had to comment those out due to recursive tailwind errors on `text-base`:

> [vite] Internal server error: You cannot `@apply` the `text-base` utility here because it creates a circular dependency.

Reply:
@gary149 do you know of any workaround?

Reply:
in those cases, could you just write the raw css like based on the [reference](https://tailwindcss.com/docs/font-size)

for example,
<img width="441" alt="Screenshot 2022-02-22 at 20 58 20" src="https://user-images.githubusercontent.com/11827707/155209392-7497c421-9908-4d8e-b9ac-6414edca4eff.png">

Reply:
It's the class name in the selector itself I think that's causing the issue.

Even this doesn't work:

```css
.prose.prose-doc .docstring .text-base p {
	/* @apply text-base; */
	font-size: 1rem;
	line-height: 1.5rem;
}

.prose.prose-doc .docstring li.text-base,
.prose.prose-doc .docstring .text-base li {
	/* @apply my-3; */
	margin-top: 0.75rem;
	margin-bottom: 0.75rem;
}

.prose.prose-doc .docstring .text-base code {
	/* @apply text-sm; */
	font-size: 0.875rem;
	line-height: 1.25rem;
}
```